graph/step.go

```diff
@@ -60,6 +63,7 @@ type Step struct {
 	IgnoreErrors                    bool `yaml:"ignoreErrors"`
 	DisableWorkingDirectoryOverride bool `yaml:"disableWorkingDirectoryOverride"`
 	Pull                            bool `yaml:"pull"`
+	EnableCache                     bool `yaml:"enableCache"`
```

Comment:
I wonder if it's really worth having this. Like if people don't want it can't they just update their YAML. I don't like the idea of adding more and more parameters to support upstream. I would prefer if they could do everything in YAML.